mopidy/core/actor.py

```diff
@@ -136,6 +138,106 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def on_start(self):
```

Comment:
We might not want this in an `on_start`, but instead have a `setup` method that our startup code calls and does a `.get()` on before starting frontends (https://github.com/mopidy/mopidy/blob/develop/mopidy/commands.py#L304). Reason for this is simply error handling and a slight design mistake in pykka. Things that happen in `on_start` can't be handled in a sane way as far as exception handling is concerned.